src/web/components/c-recipe-li.mjs

```diff
@@ -0,0 +1,197 @@
+
+        // handle click on 'disable-icon'
+        if (e.target === disableIcon) {
+            if (disableIcon.getAttribute("disabled") === "false") {
+                disableIcon.setAttribute("disabled", "true");
+                disableIcon.classList.add("disable-icon-selected");
+                this.querySelector("li.operation").classList.add("disabled");
+            } else {
+                disableIcon.setAttribute("disabled", "false");
+                disableIcon.classList.remove("disable-icon-selected");
+                this.querySelector("li.operation").classList.remove("disabled");
+            }
+
+            this.app.progress = 0;
+            window.dispatchEvent(this.app.manager.statechange);
+        }
+
+        // handle click on 'breakpoint-icon'
+        if (e.target === breakpointIcon) {
+            if (breakpointIcon.getAttribute("break") === "false") {
+                breakpointIcon.setAttribute("break", "true");
+                breakpointIcon.classList.add("breakpoint-selected");
+            } else {
+                breakpointIcon.setAttribute("break", "false");
+                breakpointIcon.classList.remove("breakpoint-selected");
+            }
+
+            window.dispatchEvent(this.app.manager.statechange);
+        }
+    }
+
+    /**
+     * Remove this operation from the recipe list
+     *
+     * @fires Manager#statechange
+     */
+    removeOperation() {
+        this.remove();
+        log.debug("Operation removed from recipe");
+        window.dispatchEvent(this.app.manager.statechange);
+        window.dispatchEvent(this.app.manager.operationremove);
+    }
+
+    /**
+     * Build the ingredient list item
+     */
+    build() {
+        const li = document.createElement("li");
+        li.classList.add("operation");
+        li.setAttribute("data-name", this.name);
+
+        const titleDiv = document.createElement("div");
+        titleDiv.classList.add("op-title");
+        titleDiv.innerText = this.name;
+
+        const ingredientDiv = document.createElement("div");
+        ingredientDiv.classList.add("ingredients");
+
+        li.appendChild(titleDiv);
+        li.appendChild(ingredientDiv);
+
+        for (let i = 0; i < this.ingredients.length; i++) {
+            ingredientDiv.innerHTML += (this.ingredients[i].toHtml());
+        }
+
+        const icons = this.buildIcons();
+
+        const clearfixDiv = document.createElement("div");
+
+        if (this.flowControl) {
+            li.classList.add("flow-control-op");
+        }
+
+        if (this.manualBake && this.app.autoBake_) {
+            this.manager.controls.setAutoBake(false);
+            this.app.alert("Auto-Bake is disabled by default when using this operation.", 5000);
+        }
+
+        li.appendChild(icons);
+        li.appendChild(clearfixDiv);
+
+        this.appendChild(li);
+    }
+
+    /**
+     * Build the icons ( disable and breakpoint / pause )
```

Comment:
I haven't paid a lot of attention to the other annotations, but strictly speaking you should probably add an @returns annotation here as well, like:
`* @returns {HTMLElement}`